src/modules/fancyzones/lib/FancyZonesData.cpp

```diff
@@ -60,6 +59,94 @@ namespace
 
         return tmpDirPath;
     }
+
+    bool DeleteRegistryKey(HKEY hKeyRoot, LPTSTR lpSubKey)
+    {
+        // First, see if we can delete the key without having
+        // to recurse.
+        auto result = RegDeleteKey(hKeyRoot, lpSubKey);
```

Comment:
Personally, I don't think we need separate variable to store result, we can always just do:
```
if (RegDeleteKey(hKeyRoot, lpSubKey) == ERROR_<WHATEVER>) {
}
```
Same thing on several places bellow.